ioquake3

```diff
@@ -22,6 +23,8 @@ EOM
 # Ensure ioquake3 reaches our pak-files and configs.
 ln -ns "${XDG_DATA_HOME}" "${HOME}/.q3a"
```

Comment:
To be honest, I still do not like this solution because it means unnecessary symlink creation every time the Q3A Flatpak is started (because the symlink is not persistent and cannot be). The ".q3a" directory if displayed in the Zenity dialog should probably be fine for most users (even if it is a hidden directory) since they will just copy it from the dialog and paste it to their file manager. But this is up to you, both solutions have their upsides and downsides.